pex/commands/bdist_pex.py

```diff
@@ -55,7 +55,7 @@ def run(self):
 
     reqs = [package_dir] + reqs
 
-    with ENV.patch(PEX_VERBOSE=str(options.verbosity)):
+    with ENV.patch(PEX_VERBOSE=str(options.verbosity), PEX_ROOT=options.pex_root):
```

Comment:
what happens if `options.pex_root` is `None` here? does that clobber the default `PEX_ROOT`?